pallets/marketplace/src/payments.rs

```diff
@@ -21,6 +21,67 @@ impl<Asset> AssetBarrier<Asset> for () {
     }
 }
 
+pub(crate) type RewardFor<T> = <<T as Config>::RewardManager as RewardManager<T>>::Reward;
```

Comment:
This should probably be moved to `types.rs`

```suggestion
pub type RewardFor<T> = <<T as Config>::RewardManager as RewardManager<T>>::Reward;
```